src/components/Form/FormWrapper.tsx

```diff
@@ -110,7 +110,18 @@ function FormWrapper({
         focusInput?.focus?.();
     }, [errors, formState?.errorFields, inputRefs]);
 
-    const {paddingBottom} = useSafeAreaPaddings(true);
+    // If either of `addBottomSafeAreaPadding` or `shouldSubmitButtonStickToBottom` is explicitly set,
+    // we expect that the user wants to use the new edge-to-edge bottom safe area padding handling.
```

Comment:
might be just naming but I find it confusing that if either of the variables is defined `enableEdgeToEdgeBottomSafeAreaPadding` is false. But the comment kinda suggest we would not use edge-to-edge if any is set

Reply:
No you're right, the condition is wrong. I incorrectly applied De Morgan's Law here. My brain somehow stopped working here haha...

thanks for the catch! Now the condition reads the same as the comment